provider-registry.yaml

```diff
@@ -10,4 +10,5 @@ providers:
   - url: git+https://github.com/i-am-bee/beeai@agents-v0.0.33#path=/agents/community/langgraph-agents
   - url: git+https://github.com/i-am-bee/beeai@agents-v0.0.33#path=/agents/community/aider-agent
   - url: git+https://github.com/i-am-bee/beeai@agents-v0.0.33#path=/agents/community/autogen-agents
+  - url: git+https://github.com/psschwei/beeai@add-hf-research-agent#path=agents/community/open-deep-research-agent
```

Comment:
Please use the next higher `agents-v...` version, we will create a tag once it's merged into main.

Also, since this becomes part of the registry, the `beeai-provider.yaml` must not use local package path (`file://`).